tuist/tuist
# Include `XCTVapor` in a predefined list of testing packages

### Short description 📝

Tuist includes a list of well-known packages that should have enabled test search paths when building dependencies. Vapor’s testing package `XCTVapor` is currently unlisted. This PR adds `XCTVapor` to the list.

### How to test the changes locally 🧐

Create a Tuist project that uses Vapor and add a testing target in the Project manifest with the following dependency: `.external(name: "XCTVapor”)`.  After the project is generated it should build.

### Contributor checklist ✅


### Reviewer checklist ✅
